samples/cellular/lwm2m_client/sample_description.rst

```diff
@@ -489,7 +490,26 @@ Moreover, the sample also provides the following files for LwM2M 1.1 features:
 
 For further information about the test cases, see `Enabler Test Specification (Interoperability) for Lightweight M2M`_.
 
-You can configure the sample either by editing the :file:`prj.conf` file and the relevant overlay files, or through menuconfig or guiconfig.
+
```

Comment:
@umapraseeda Is it possible to add some sub header about: Fota external configuration?